config/locales/en.yml

```diff
@@ -1194,10 +1196,23 @@ en:
       keywords:
         one: "%{count} keyword"
         other: "%{count} keywords"
+      statuses:
+        one: "%{count} post"
+        other: "%{count} posts"
+      statuses_long:
+        one: "%{count} individual post hidden"
+        other: "%{count} individual posts hidden"
       title: Filters
     new:
       save: Save new filter
       title: Add new filter
+    statuses:
+      back_to_filter: Back to filter
+      batch:
+        remove: Remove from filter
+      index:
+        hint: This filter applies to select individual posts regardless of other criteria. You can add more posts to this filter from the Web interface.
```

Comment:
I don't think we capitalize the "web" anywhere else... It's not a name.

Reply:
We actually do in “Web browser” (and when displaying that a post has been authored in the web app, we display “Web”) but that's the exception. I'll change that.

Reply:
Okay, but those are start of a sentence situation (I believe)